extensions/amp-social-share/1.0/amp-social-share.css

```diff
@@ -65,13 +65,13 @@
 /* line Styling */
 .amp-social-share-line {
   color: #fff;
-  background-color: #52b448;
+  background-color: #00b900;
 }
 
 /* SMS Styling */
 .amp-social-share-sms {
   color: #fff;
-  background-color: #ca2b63;
+  background-color: #000000;
```

Comment:
Is it really all 0? I almost never see that.

Reply:
Right, Zewen, our UX designer has come up with a new set of SVG icons.  For SMS specifically, we've updated the color to black.  I will add you to the email chain for context!

Reply:
Ok. A bit unusual, but i'll take it.